pkg/virtctl/vmexport/vmexport.go

```diff
@@ -347,7 +347,7 @@ func (c *command) initVMExportInfo(vmeInfo *VMExportInfo) error {
 	vmeInfo.ExportSource = getExportSource()
 	vmeInfo.OutputFile = outputFile
 	// User wants the output in a file, create
-	if outputFile != "" {
+	if outputFile != "" && outputFile != "-" {
```

Comment:
you don't allow outputFile to be "" so you dont need to check it right?
what about someone explicitly doing --outputFile=stdout? we dont want that to go to stdout? is it not acceptable?